schema/artwork/index.js

```diff
@@ -120,6 +122,21 @@ const ArtworkType = new GraphQLObjectType({
           return size ? _.take(images, size) : images;
         },
       },
+      related: {
+        type: RelatedType,
+        description: 'Returns the associated Fair or Sale',
+        resolve: ({ id }) => {
+          const options = { artwork: [id], active: true, size: 1 };
+          return Promise.all([
+            gravity('related/fairs', options),
+            gravity('related/sales', options),
+          ]).then(([fairs, sales]) => {
+            fairs.map(fair => fair.related_type = 'Fair'); // eslint-disable-line no-param-reassign
```

Comment:
Yeah; I guess i should be doing a `_.assign({}, fair, { related_type: 'Fair' })` :palm_tree: